snapcraft/tests/test_yaml.py

```diff
@@ -63,6 +63,49 @@ def test_config_loads_plugins(self, mock_get_part, mock_loadPlugin):
         self.assertFalse(mock_get_part.called)
 
     @unittest.mock.patch('snapcraft.yaml.Config.load_plugin')
+    def test_config_works_with_skills(self, mock_loadPlugin):
+        self.make_snapcraft_yaml("""name: test
+version: "1"
+summary: test
+description: test
+
+parts:
+  part1:
+    plugin: go
+    stage-packages: [fswebcam]
+
+apps:
+   app1:
+     command: runme
+     uses:
```

Comment:
yes, that is why the test is called *with skills*

you don't have to face angry users, I do :wink: